tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -332,6 +335,18 @@
         </convert>
     </target>
 
+    <target name="copy-cldrtestdata" depends="init-args">
```

Comment:
Surely copy should depend on clean, otherwise you risk inconsistent test data (which is a confusing state to be in).

Reply:
Thanks, good point - done!